relay-general/src/store/transactions.rs

```diff
@@ -93,9 +93,9 @@ pub fn is_high_cardinality_sdk(event: &Event) -> bool {
         return true;
     }
 
-    let is_http_method_options = event.get_tag_value("http.method") == Some("OPTIONS");
+    let http_method = event.request.value().and_then(|r| r.method.as_str());
     if sdk_name == "sentry.javascript.node"
-        && is_http_method_options
+        && http_method == Some("OPTIONS")
```

Comment:
There's a normalization that doesn't yet run before metrics extraction that will uppercase the method: https://github.com/getsentry/relay/blob/ed3a521f773ecd4bf9a2aefd7af80080d56d0841/relay-general/src/store/normalize/request.rs#L81

I think Express is sending lowercase, so let's use `unwrap_or_default()` and [`eq_ignore_ascii_case`](https://doc.rust-lang.org/stable/std/primitive.str.html#method.eq_ignore_ascii_case) here